LevelSpaceXWGUI.nlogo

```diff
@@ -572,7 +589,7 @@ to-report make-variadic-task [astring args]
   set astring add-spaces astring
   foreach string:rex-split astring "\\s" [
     ifelse member? ? args[
-      set sb lput (word "(item " (position ? args) " ?)") sb
+      set sb lput (word "(item " (position ? args) " ?" arg-num ")") sb
```

Comment:
In that thread about tasks vs strings, this is pretty much exactly the counter-example to tasks-only that I was thinking...

string-evaluation: a poor man's macros.